.github/workflows/ci.yaml

```diff
@@ -77,25 +79,96 @@ jobs:
         run: build cmake
       - name: Build packages
         run: build package
+      - name: Upload rpm package
+        uses: actions/upload-artifact@v3
+        with:
+          name: sysdig-dev-linux-amd64.rpm
+          path: |
+            /build/release/sysdig-*.rpm
+      - name: Upload deb package
+        uses: actions/upload-artifact@v3
+        with:
+          name: sysdig-dev-linux-amd64.deb
+          path: |
+            /build/release/sysdig-*.deb
+      - name: Upload tar.gz package
+        uses: actions/upload-artifact@v3
+        with:
+          name: sysdig-dev-linux-amd64.tar.gz
+          path: |
+            /build/release/sysdig-*.tar.gz      
 
   build-sysdig-linux-arm64:
+    needs: builder
```

Comment:
I know this is slow as hell but we need to distribute static artifacts built with bundled deps :) otherwise it is pretty useless to distribute them!